src/test/scala/org/constellation/E2E.scala

```diff
@@ -17,12 +15,12 @@ trait E2E extends AsyncFlatSpecLike with Matchers with BeforeAndAfterAll with Be
   implicit val materializer: ActorMaterializer = ActorMaterializer()
 //  implicit val timeout: Timeout = Timeout(90, TimeUnit.SECONDS)
 
-  val tmpDir = "tmp"
+  val tmpDir = "/tmp"
 
-  override def beforeAll(): Unit =
-    // Cleanup DBs
-    //Try{File(tmpDir).delete()}
+  override def beforeAll(): Unit = {
     Try { File(tmpDir).createDirectories() }
+    Try { File(tmpDir).delete() }
```

Comment:
🤔 

Why we create and then delete?

If it is because `delete` throws if directory doesn't exist then just `File(tmpDir).exists` before

Reply:
Looks like I switched them accidentally